manifest.php

```diff
@@ -29,7 +29,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '16.2.1',
+    'version'     => '16.2.2',
```

Comment:
Since the change is heavier that a simple fix, could you increase the minor version instead?